k8s-install/scripts/install-cni.sh

```diff
@@ -129,7 +129,7 @@ sed -i s~__ETCD_ENDPOINTS__~${ETCD_ENDPOINTS:-}~g $TMP_CONF
 # Move the temporary CNI config into place.
 FILENAME=${CNI_CONF_NAME:-10-calico.conf}
 mv $TMP_CONF /host/etc/cni/net.d/${FILENAME}
-echo "Wrote CNI config: $(cat /host/etc/cni/net.d/${FILENAME})"
+echo "Wrote CNI config: $(cat /host/etc/cni/net.d/${FILENAME} | sed '/k8s_auth_token/s/\: \"[a-zA-Z0-9\_\-\.\-]*/\: \"\*\*\*\*\*/g')"
```

Comment:
You could simplify the sed a bit with `sed '/k8s_auth_token/s/\: \"[^"]*/\: \"*****/g'`
Might as well just match on things that are not `"` and in the replacement half you don't have to escape the `*`.